maskrcnn_benchmark/data/datasets/evaluation/__init__.py

```diff
@@ -16,12 +17,20 @@ def evaluate(dataset, predictions, output_folder, **kwargs):
         evaluation result
     """
     args = dict(
-        dataset=dataset, predictions=predictions, output_folder=output_folder, **kwargs
+        dataset=dataset,
+        predictions=predictions,
+        output_folder=output_folder,
+        **kwargs
     )
     if isinstance(dataset, datasets.COCODataset):
         return coco_evaluation(**args)
     elif isinstance(dataset, datasets.PascalVOCDataset):
         return voc_evaluation(**args)
+    elif isinstance(dataset, datasets.AbstractDataset):
```

Comment:
If no specific evaluation tool was assigned to `dataset` have COCO eval as fallback option